Makefile

```diff
@@ -1,16 +1,23 @@
-export RELEASE_BRANCH?=1-18
-export RELEASE?=1
-export DEVELOPMENT?=false
-export AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
-export AWS_REGION?=us-west-2
-export IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+RELEASE_BRANCH?=1-18
+DEFAULT_RELEASE=$(shell cat $(BASE_DIRECTORY)/release/$(RELEASE_BRANCH)/RELEASE)
+ifneq ("$(PULL_NUMBER)","")
+	RELEASE?=$(shell expr $(DEFAULT_RELEASE) \* 10000 + $(PULL_NUMBER))
```

Comment:
As discussed offline, let's use commit hash instead of this being dependent on the PR number so that it is cleaner and more logical.